packages/generator/src/GenerateContext/enhanceOntology.ts

```diff
@@ -18,15 +18,25 @@ import type { WireOntologyDefinition } from "../WireOntologyDefinition.js";
 import { EnhancedOntologyDefinition } from "./EnhancedOntologyDefinition.js";
 
 export function enhanceOntology(
-  sanitized: WireOntologyDefinition,
-  ontologyApiNamespace: string | undefined,
-  apiNamespacePackageMap: Map<string, string>,
-  importExt: string,
+  {
```

Comment:
converted to options style because it made reading it easier than all the undefineds.